app/javascript/src/actions/utils/crud.js

```diff
@@ -1,36 +1,45 @@
-import axios from 'axios';
-import { push } from 'react-router-redux';
+import axios from "axios";
+import { push } from "react-router-redux";
 
-const crud = request => (dispatch) => {
+//const { INTERNAL_API_KEY } = process.env;
+const INTERNAL_API_KEY = "hellopass";
```

Comment:
This needs to be removed. The key cannot be public and it must come from the env vars.